README.md

```diff
@@ -69,6 +69,18 @@ python3 MaxText/decode.py MaxText/configs/base.yml run_name=$YOUR_JOB_NAME
 ```
 Be aware, these decodings will be random. To get high quality decodings you need pass in a checkpoint, typically via the `load_parameters_path` argument.
 
+
+#### Running on NVIDIA GPUs
+1. `maxtext_dependencies.Dockerfile` can be used to build a container with the required dependencies.
```

Comment:
Total nit, but can you just say more clearly what the user should do? Is it still `bash docker_build_dependency_image.sh` or something else?

Reply:
@rwitten @michelle-yooh I have made the changes to make it more clear.
@michelle-yooh, will you be merging your PR as well (#265)?

Reply:
yes, I'll be merging #265 after this